README.md

```diff
@@ -27,6 +27,12 @@ angular.module('myApp', ['angular-mailcheck']);
 
 ```html
 <input type="email" mailcheck>
+
+<!-- You can add suggestion and nope text -->
+<input type="email" mailcheck mailcheck-suggestion-text="suggestionText" mailcheck-nope-text="nopeText">
+
+<!-- When using angular-translate -->
+<input type="email" mailcheck mailcheck-suggestion-text="'EMAIL_SUGGESTION_OFFER' | translate" mailcheck-nope-text="'EMAIL_SUGGESTION_PASS' | translate">
 ```
```

Comment:
Thanks. Documentation could be something like:

---
 ```html
<input type="email" mailcheck>
```

You can add suggestion and nope texts as strings:
```html
<input type="email" mailcheck mailcheck-suggestion-text="'Did you mean:'" mailcheck-nope-text="'Nope'">
```
...or as scope variables: 
```html
<input type="email" mailcheck mailcheck-suggestion-text="suggestionText" mailcheck-nope-text="nopeText">
```

...or use [Angular-Translate](https://angular-translate.github.io/):
```html
<input type="email" mailcheck mailcheck-suggestion-text="'EMAIL_SUGGESTION_OFFER' | translate" mailcheck-nope-text="'EMAIL_SUGGESTION_PASS' | translate">
```
---
What do you think?